### DISCUSSION TITLE
月間トレンド記事一覧が途中で切れているように見える

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#495] [要望・アイデア・不具合]
[月間トレンド記事一覧](https://qiita.com/Qiita/items/616e8f6d4f69bd582ab5) の 2023年6月15日および2023年7月1日の更新分について、途中で切れているような見え方をしています。

![image](https://github.com/increments/qiita-discussions/assets/15214393/0964fe33-353d-422f-9c07-03ce5de9312b)

マークダウンで確認すると、以下の `<script setup>` 以降の表示と、その後の 11 記事が表示されていないようです。

```
### [Vue3の衰退を招いたのは<script setup>とCompositionAPIかもしれない という考察](https://qiita.com/fruitriin/items/81691ce68cf3678f3bda)

タグ: [ポエム](https://qiita.com/tags/%E3%83%9D%E3%82%A8%E3%83%A0), [Vue.js](https://qiita.com/tags/vue.js), [Vue3](https://qiita.com/tags/vue3)
by [fruitriin](https://qiita.com/fruitriin)
:like: +239
```

環境は、Windows11 + Chrome です

**Comment by wataru86** [comment]
@yoshi389111

ご報告ありがとうございます！:pray:
全ての記事が表示されるように修正しました！
また、来月以降の更新の際に同様の問題が発生しないよう、根本的な解決に向けた修正も行っていきます！
